lib/share_notify/graph.rb

```diff
@@ -26,8 +26,7 @@ def creative_work
       add_property(creative_work, :date_published, push_doc.date_published)
       add_property(creative_work, :date_updated, push_doc.providerUpdatedDateTime)
       add_property(creative_work, :extra, push_doc.extra)
-
-      creative_work[:is_deleted] = true if push_doc.deleted?
+      creative_work[:is_deleted] = push_doc.deleted? ? true : false
```

Comment:
`.deleted?` is going to return `true` or `false`, so the ternary is unnecessary. 
``` ruby
creative_work[:is_deleted] = push_doc.deleted?
```
ought to suffice.